src/v2/coins/abstractUtxoCoin.js

```diff
@@ -261,6 +261,9 @@ class AbstractUtxoCoin extends BaseCoin {
             // the address was found, but not on the wallet, which simply means it's external
             debug('Address %s presumed external', currentAddress);
             return _.extend({}, currentOutput, { external: true });
+          } else if (e.message.includes('address validation failure: invalid chain') && currentAddress === txParams.changeAddress) {
```

Comment:
Since we are throwing a nice custom error type, you should check the error type instead of the message.

Reply:
looking at verifyAddress, and we do throw the error type InvalidAddressVerificationObjectPropertyError for this case, but we also throw that error type if the coinSpecific param is not an object. So maybe we do want to check for the error message?

Reply:
If these error cases should be distinguished, then a separate error type should be introduced. Checking error message text is bad form, and we should really not be doing this. I'll leave it to you to decide as to whether or not these cases need to be distinguished.